cypress/commons/actions/generic/Scenarios.js

```diff
@@ -243,14 +243,35 @@ function createScenario(scenarioName, isMaster, datasetOrMasterName, runTemplate
   });
 }
 
-function validateScenario() {
-  return getScenarioValidateButton().click();
+function validateScenario(scenarioId) {
+  const validateScenarioAlias = api.interceptUpdateScenario(scenarioId);
+  const getScenarioAlias = api.interceptGetScenario(scenarioId);
+
+  getScenarioValidateButton().click();
+  Scenarios.getScenarioValidationStatusLoadingSpinner().should('be.visible');
+
+  api.waitAlias(getScenarioAlias);
+  return api.waitAlias(validateScenarioAlias);
```

Comment:
`return` may not be necessary here, and isn't consistent with the other functions below